src/Model/Gallery.php

```diff
@@ -106,9 +106,9 @@ final public function getDefaultFormat(): string
         return $this->defaultFormat;
     }
 
-    final public function setGalleryItems(Collection $galleryItems): void
+    final public function setGalleryItems(iterable $galleryItems): void
```

Comment:
Not sure about that, an array is iterable right? But if I pass array, that clear method wont work

Reply:
The `clear()` method will be executed on `ArrayCollection` (the value set for the `galleryItems` property at `__construct()`), not in the array provided as argument.

Reply:
Should we add a test ?

Reply:
I think these changes are covered by PHPStan, but I can add a test if you consider it is required.

Reply:
Indeed i think you're right